### PR TITLE
Update home.ts

### PR DESCRIPTION
**Pull request by thednp**
Add VanJS Feather

### CODE REVIEW

File: sitegen/home.ts [hunk 1]
```diff
@@ -138,6 +138,11 @@ van.add(document.body, Counter())
           td("An SPA framework add-on for ", VanJS()),
           td(Link("b-rad-c", "https://github.com/b-rad-c")),
         ),
+        tr(
+          td(Link("VanJS Feather", "https://thednp.github.io/vanjs-feather/")),
```
Comment (Tao-VanJS) [inline sitegen/home.ts:142]: Can you put some instructions on how to use your library in the website? If the github repo servers better for this purpose, you can put the link to the github repo here.

**Review by Tao-VanJS — CHANGES_REQUESTED** [review]

File: sitegen/home.ts (hunk 1 above)
Comment (thednp) [reply sitegen/home.ts:142]: That's the plan, to include instructions.

Comment (Tao-VanJS) [reply sitegen/home.ts:142]: Sure. I will merge this PR once the instructions are there.

Comment (thednp) [reply sitegen/home.ts:142]: https://thednp.github.io/vanjs-feather/

**Pull request merged**